package.json

```diff
@@ -1,41 +1,42 @@
-{
-  "name": "aria-at-automation-harness",
-  "version": "0.0.0",
-  "private": true,
-  "type": "module",
-  "bin": {
-    "aria-at-harness-agent": "./bin/agent.js",
-    "aria-at-harness-host": "./bin/host.js"
-  },
-  "devDependencies": {
-    "ava": "^4.1.0",
-    "husky": "^7.0.2",
-    "jsdoc": "^3.6.7",
-    "lint-staged": "^11.2.3",
-    "prettier": "^2.4.1"
-  },
-  "dependencies": {
-    "express": "^4.17.1",
-    "node-fetch": "^3.3.2",
-    "selenium-webdriver": "^4.1.0",
-    "ws": "^8.3.0",
-    "yargs": "^17.2.1"
-  },
-  "engines": {
-    "node": ">=16.0.0"
-  },
-  "homepage": "https://github.com/bocoup/aria-at-harness",
-  "scripts": {
-    "build": "npm run build:docs",
-    "build:docs": "jsdoc src -r -d docs",
-    "test": "npm run test:unit",
-    "test:unit": "ava",
-    "postinstall": "husky install"
-  },
-  "lint-staged": {
-    "*.{js,css,md}": "prettier --write"
-  },
-  "ava": {
-    "timeout": "1m"
-  }
-}
+{

```

Comment:
really unsure why `npm install --save minimatch` generated this big of a diff

Reply:
It switched the file to Windows line endings. Any chance you ran the command on a Windows machine?

Reply:
I'll try to get the cr/lf noise reduced on this json before we land